### DISCUSSION TITLE
How to write in color?

### DISCUSSION THREAD

**Discussion by iliakan** [sourcelair/xterm.js#4416] [Q&A] [answered]
I tried to insert ANSI-color sequences from bash, listed at https://gist.github.com/JBlond/2fea43a3049b38287e5e9cefc87b2124, but the output is different from expected.

Is it me doing something wrong? Anything special regarding `term.write` and colors, maybe special doc somewhere? ;)

**Comment by jerch** [comment, marked as answer]
And how was your output different from your expectations? If the colors dont show up, you prolly have a mistake in the sequences yeah.
(Note the "\e" in the examples above is the ESCAPE character, which you have to code in a way Javascript understands, eg. as "\x1b")

**Reply by iliakan** [reply]
Thank you, I changed `\e` into `\x1b`, and it worked ;)
